src/lib/embed.js

```diff
@@ -23,14 +29,20 @@ const oEmbedParameters = [
     'maxheight',
     'maxwidth',
     'muted',
+    'play_button_position',
     'playsinline',
     'portrait',
+    'progress_bar',
+    'quality_selector',
```

Comment:
How come this isn't just `quality`?

Reply:
Good question. There is an existing `quality` embed parameter [here](https://developer.vimeo.com/player/sdk/embed), which controls the playback quality of the video. `quality_selector` more explicitly indicates the UI element that this parameter controls.

Reply:
Gotcha, makes sense